tests/test_wave_05.py

```diff
@@ -13,11 +13,11 @@ def test_new_genre_rec():
     # Assert
     for rec in recommendations:
         assert rec not in sonyas_data["watched"]
-    assert len(recommendations) == 1
+    assert len(recommendations) == 1 #SHOULD THIS BE 2???
```

Comment:
This should be 1.
We can see that Sonya's data looks like this:
```
USER_DATA_4 = {
    "watched": [
        FANTASY_1b,
        FANTASY_2b,
        FANTASY_3b,
        ACTION_1b,
        INTRIGUE_1b,
        INTRIGUE_2b
        ],
    "friends":  [
        {
            "watched": [
                FANTASY_1b,
                FANTASY_3b,
                FANTASY_4b,
                HORROR_1b,
            ]
        },
        {
            "watched": [
                FANTASY_1b,
                ACTION_1b,
                INTRIGUE_1b,
                INTRIGUE_3b,
            ]
        }
    ]
}
```
We can see that Sonya's most watched genre is Fantasy and she has seen the following movies
```
FANTASY_1b,
FANTASY_2b,
FANTASY_3b
```
If we look at the fantasy movies her friends have seen, we get this list:
```
FANTASY_1b,
FANTASY_3b,
FANTASY_4b
```
The only fantasy movie her friends have seen that she hasn't is `FANTASY_4b` meaning our recommended list should be a length of 1.